### DISCUSSION TITLE
[WASM] Can't include file as binary data in asm code

### DISCUSSION THREAD

**Discussion by 8bitbubsy** [open-watcom/open-watcom-v2#818] [Q&A]
I need to include a raw file as binary data in my 32-bit i386 assembly code, but when I try INCBIN, all I get is "syntax error".
Is there really no INCBIN directive in the wasm assembler?

Sorry if this is written in the manual, but I can't really find any information about this!

**Comment by jmalak** [comment]
Yes, INCBIN directive is not supported by WASM.
It is also not supported by similar assemblers as MASM and TASM.
